src/instantx.cpp

```diff
@@ -1153,10 +1153,7 @@ bool CTxLockVote::IsFailed() const
 
 bool COutPointLock::AddVote(const CTxLockVote& vote)
 {
-    if(mapMasternodeVotes.count(vote.GetMasternodeOutpoint()))
-        return false;
-    mapMasternodeVotes.insert(std::make_pair(vote.GetMasternodeOutpoint(), vote));
-    return true;
+    return mapMasternodeVotes.emplace(std::make_pair(vote.GetMasternodeOutpoint(), vote)).second;
```

Comment:
what does the `.second` do in this instance?

Reply:
Ahh, that makes more sense. I guess I missed the first part of the return doc when I read it a few minutes ago. Thanks

Reply:
No need for `make_pair` here with `emplace`.